.agent-source/skills/create-pr/SKILL.md

```
@@ -172,21 +170,19 @@ Draft the PR body based on the commits and diff (`git log origin/master..HEAD` a
 - [ ] one checkbox per changed service or flow
 ```
 
-Then tell the user:
-
-> "The branch is pushed. Run the command below directly in your terminal (the `!` prefix bypasses the hook):"
+Then create the PR directly:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)